deploy/shakenfist_ci/tests/test_images.py

```diff
@@ -8,7 +10,26 @@ def test_cache_image(self):
 
         self.system_client.cache_image(url)
         image_urls = []
-        for image in self.system_client.get_image_meta():
+        for image in self.system_client.get_images():
             image_urls.append(image['url'])
 
         self.assertIn(url, image_urls)
+
+        # It would be better if this used a get_image() call, but that doesn't
+        # exist at the moment.
+        cache = {}
+        start_time = time.time()
+        while time.time() - start_time < 300:
+            cache = {}
+            for img in self.system_client.get_images():
+                cache.setdefault(img['url'], [])
+                cache[img['url']].append(img)
+
+            self.assertEqual(1, len(cache.get(url)))
```

Comment:
Would `self.assertIn(url, cache)` be better?

Reply:
Done.